go.mod

```diff
@@ -3,15 +3,16 @@ module google.golang.org/grpc
 go 1.11
 
 require (
+	cloud.google.com/go v0.63.0 // indirect
 	github.com/cncf/udpa/go v0.0.0-20200629203442-efcf912fb354
 	github.com/envoyproxy/go-control-plane v0.9.7
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/protobuf v1.4.2
-	github.com/google/go-cmp v0.5.0
+	github.com/google/go-cmp v0.5.1
```

Comment:
Is there a reason for updating all these dependencies?

If not, I'd rather remove them from this PR and update our dependencies in a separate change.

Reply:
I don't remember updating any of them manually. Let me see what's going on.

Reply:
`go get -u` will do this if you ran it.